libs/openFrameworks/graphics/ofTrueTypeFont.cpp

```diff
@@ -17,70 +17,70 @@
 #include FT_TRIGONOMETRY_H
 
 using std::max;
-using std::vector;
-using std::string;
 using std::min;
+using std::string;
+using std::vector;
 
-const ofUnicode::range ofUnicode::Space {32, 32};
-const ofUnicode::range ofUnicode::IdeographicSpace {0x3000, 0x3000};
-const ofUnicode::range ofUnicode::Latin {32, 0x007F};
-const ofUnicode::range ofUnicode::Latin1Supplement {32,0x00FF};
-const ofUnicode::range ofUnicode::LatinA {0x0100,0x017F};
-const ofUnicode::range ofUnicode::Greek {0x0370, 0x03FF};
-const ofUnicode::range ofUnicode::Cyrillic {0x0400, 0x04FF};
-const ofUnicode::range ofUnicode::Arabic {0x0600, 0x077F};
-const ofUnicode::range ofUnicode::ArabicSupplement {0x0750, 0x077F};
-const ofUnicode::range ofUnicode::ArabicExtendedA {0x08A0, 0x08FF};
-const ofUnicode::range ofUnicode::Devanagari {0x0900, 0x097F};
-const ofUnicode::range ofUnicode::HangulJamo {0x1100, 0x11FF};
-const ofUnicode::range ofUnicode::VedicExtensions {0x1CD0, 0x1CFF};
-const ofUnicode::range ofUnicode::LatinExtendedAdditional {0x1E00, 0x1EFF};
-const ofUnicode::range ofUnicode::GreekExtended {0x1F00, 0x1FFF};
-const ofUnicode::range ofUnicode::GeneralPunctuation {0x2000, 0x206F};
-const ofUnicode::range ofUnicode::SuperAndSubScripts {0x2070, 0x209F};
-const ofUnicode::range ofUnicode::CurrencySymbols {0x20A0, 0x20CF};
-const ofUnicode::range ofUnicode::LetterLikeSymbols {0x2100, 0x214F};
-const ofUnicode::range ofUnicode::NumberForms {0x2150, 0x218F};
-const ofUnicode::range ofUnicode::Arrows {0x2190, 0x21FF};
-const ofUnicode::range ofUnicode::MathOperators {0x2200, 0x22FF};
-const ofUnicode::range ofUnicode::MiscTechnical {0x2300, 0x23FF};
-const ofUnicode::range ofUnicode::BoxDrawing {0x2500, 0x257F};
-const ofUnicode::range ofUnicode::BlockElement {0x2580, 0x259F};
-const ofUnicode::range ofUnicode::GeometricShapes {0x25A0, 0x25FF};
```

Comment:
a lot of formatting changes here hence the stale conflict. if you wanna figure it out